src/test/java/org/broadinstitute/hellbender/tools/walkers/genotyper/AlleleSubsettingUtilsUnitTest.java

```diff
@@ -56,7 +56,9 @@ public void testUpdatePLsAndADData(final VariantContext originalVC,
         final List<Allele> AA = Arrays.asList(Aref, Aref);
         final List<Allele> AC = Arrays.asList(Aref,C);
         final List<Allele> CC = Arrays.asList(C,C);
+        final List<Allele> CG = Arrays.asList(C,G);
         final List<Allele> AG = Arrays.asList(Aref,G);
+        final List<Allele> GG = Arrays.asList(G,G);
```

Comment:
The genotypes around line 74 have bad ADs too -- `Genotype acGT` and `Genotype ccGT`

Reply:
done